frontend/.env.sample

```
@@ -1 +1,2 @@
-VITE_CLERK_PUBLISHABLE_KEY=
\ No newline at end of file
+VITE_CLERK_PUBLISHABLE_KEY=
+VITE_BACKEND_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_BACKEND_URL key should go before the VITE_CLERK_PUBLISHABLE_KEY key

(UnorderedKey)